recipes/go4/recipe.yaml

```diff
@@ -0,0 +1,100 @@
+context:
+  name: Go4
+  version: '6.4.1'
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/gsi-ee/go4//releases/download/${{ version }}/go4-${{ version }}.tar.gz
+  sha256: d4c4409b4a86e8686ac19a040824fc37f8ad9bdeef27545cb405b777025e12ee
+
+build:
+  number: 0
+  skip:
+    - win
+  script:
+    - if: unix
+      then:
+      - "${RECIPE_DIR}/build.sh"
+  dynamic_linking:
+    overdepending_behavior: "error"
+    overlinking_behavior: "error"
+    missing_dso_allowlist:
+      - if: osx
+        then:
+          - /usr/lib/libSystem.B.dylib
```

Comment:
```suggestion
```
These shouldn't be necessary.  If you're getting errors relating to linking to `libc` (which is part of `libSystem.B.dylib` on macOS), then we need to investigate if some thing is bypassing our compiler toolchain or downloading binaries.